### PR TITLE
Meetings set local video quality

### PR DESCRIPTION
**Pull request by adamweeks**
Adds the ability to set the local video quality during a meeting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/constants.js [hunk 1]
```diff
@@ -962,3 +962,44 @@ export const MQA_STATS = {
     systemMaximumCPU: 0
   }
 };
+
+export const VIDEO_RESOLUTIONS = {
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/constants.js:966]: Should we have a config for what max resolution we should receive

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 2]
```diff
@@ -2773,4 +2774,35 @@ export default class Meeting extends StatelessWebexPlugin {
...
   }
+
+  /**
+   * Sets the quality of the local video stream
+   * @param {String} level {LOW|MEDIUM|HIGH}
+   * @returns {Promise}
+   */
+  setLocalVideoQuality(level) {
+    LoggerProxy.logger.log(`Meeting:index#setLocalVideoQuality --> Setting quality to ${level}`);
+
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:2785]: add a error check if sendVideo is false

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 3]
```diff
@@ -2773,4 +2774,35 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return reject('Meeting:index#sendDTMF --> cannot send DTMF, meeting does not have a connection to the "locus" call control service. Have you joined?');
   }
+
+  /**
+   * Sets the quality of the local video stream
+   * @param {String} level {LOW|MEDIUM|HIGH}
+   * @returns {Promise}
+   */
+  setLocalVideoQuality(level) {
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:2783]: Might need to rethink about this function as its doing nothing other then getUserMedia with lower resolution then doing updateMedia 

Do we still need to have different function to downgrade incoming video and keep it independent of this

Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:2783]: The scope of this PR is limited to just the resolution of the sending media. We will have another for the receive media resolution.

File: packages/node_modules/@webex/plugin-meetings/src/constants.js (hunk 1 above)
Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/constants.js:966]: All of these configs have max resolutions set, so I'm not sure what you are asking.

Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/constants.js:966]: Was talking about default config . If someone wants to start a meeting by default with lower resolution or video

Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/constants.js:966]: Good idea! I've created another task for changing settings pre-join as this one is scope towards during call changes. https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-133817

File: packages/node_modules/@webex/plugin-meetings/src/common/errors/parameter.js [hunk 4]
```diff
@@ -12,10 +12,15 @@ export default class ParameterError extends Error {
   */
   constructor(message = ERROR_DICTIONARY.PARAMETER.MESSAGE, error = null) {
     super(message);
+
+    if (Error.captureStackTrace) {
```
Comment (lalli-flores) [inline packages/node_modules/@webex/plugin-meetings/src/common/errors/parameter.js:16]: Can this change be in its own commit? since it doesn't relate to local video quality changes?

Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/common/errors/parameter.js:16]: Yes, this was actually from another branch that got in here somehow during a rebase.
